### DISCUSSION TITLE
setCD parameter not working as expected - Disable Health Checks

### DISCUSSION THREAD

**Discussion by meluvalli** [PowerDNS/pdns#13823] [Q&A] [answered]
Hello.  I have a backend DNS server and I want to disable health checks for.  I used setCD=true, but it doesn't seem to work.  I am still getting queries on my backend for health checks.  How would I disable health checks for a specific server?

This is my server line:
```
newServer({address=NextDNS_Address, tls="openssl", subjectName="dns.nextdns.io", dohPath="/acf3f1", setCD=true, name="Secure DNS NextDNS", pool="NextDNS"})
```
Any assistance would be greatly appreciated!

**Comment by meluvalli** [comment, marked as answer]
Adding healthCheckMode='up' resolved the issue.

**Comment by rgacogne** [comment]
For the record, `setCD=true` sets the Checking Disabled bit in the health-check query: `The CD (checking disabled) bit indicates in a query that non-verified data is acceptable to the resolver sending the query`. It does not disable health-checks.
